apps/design-land/src/app/modal/modal.component.ts

```diff
@@ -1,18 +1,23 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
+
+import { DaffModalService } from '@daffodil/design';
+
+import { DesignLandModalContentComponent } from './modal-content/modal-content.component';
 
 @Component({
   selector: 'design-land-modal',
   templateUrl: './modal.component.html',
   styleUrls: ['./modal.component.scss']
 })
-export class ModalComponent implements OnInit {
+export class ModalComponent {
+  modal: any;
+  constructor(private modalService: DaffModalService) {}
 
-  constructor() { }
-
-  ngOnInit() {
+  showModal() {
+    this.modal = this.modalService.open(DesignLandModalContentComponent);
   }
-  private showModal : boolean;
+
   hideModal() {
```

Comment:
This function doesn't do anything

Reply:
Fixed in b75505d2cbbe48d71a604ee3a1eb1a5c597ebc24